src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -355,7 +394,10 @@ void NSModel<SortPolicy>::Search(arma::mat&& querySet,
   if (randomBasis)
     querySet = q * querySet;
 
-  Log::Info << "Searching for " << k << " nearest neighbors with ";
+  Log::Info << "Searching for " << k;
+  if (Epsilon() != 0)
+    Log::Info << " approximate (e=" << Epsilon() << ")";
```

Comment:
I guess we could avoid the problem entirely by writing, e.g., "1.05-approximate " (where e=0.05).